server/server.js

```diff
@@ -8,9 +8,9 @@ const app = express();
 app.use(bodyparser.json());
 
 
-app.use(express.static(`${__dirname}/../client/dist`));
+app.use('/listings/:name/:id/:imagesId/', express.static(`${__dirname}/../client/dist`));
```

Comment:
The url format should be :id/:name, the way Etsy has it - and the way I have it, so it'll probably create problems in the proxy server if you expect a different format.
I'm also not sure why your basic path has the imagesId in it: you want to send back your static assets when they go to a basic listings page, right?

Reply:
took out :id